src/diagonal.works/b6/ingest/worlds.go

```diff
@@ -0,0 +1,43 @@
+package ingest
+
+import (
+	"diagonal.works/b6"
+)
+
+var DefaultWorldFeatureID = b6.FeatureID{
+	Type:      b6.FeatureTypeCollection,
+	Namespace: "diagonal.works/world",
+	Value:     0,
+}
+
+type Worlds interface {
+	FindWorld(id b6.FeatureID) MutableWorld
+}
+
+type MutableWorlds struct {
+	Base    b6.World
+	Mutable map[b6.FeatureID]MutableWorld
+}
+
+func (m *MutableWorlds) FindWorld(id b6.FeatureID) MutableWorld {
+	if !id.IsValid() {
+		id = DefaultWorldFeatureID
```

Comment:
i m not sure i like this default id and fallback / y do we want this ?

Reply:
I don't know whether we'll have it forever - but right now, we have a --world command line argument, so effectively this ID maps to that world. In future, we may end up with a config file mapping IDs to specific indices, but I'm not sure what that will look like yet, nor whether it'll live in the open source codebase or not, so I'm sticking with this simple approach for the moment.